### ISSUE TITLE
Link to accessibility statement and email link to report problems?

### ISSUE THREAD

**Issue by ydmitry** [jossef/open-accessability#12]
If I am not wrong, Israeli law about accessibility contains a paragraph about text "Accessibility statement" and person should be available to write an email letter to report problems about accessibility (and you have 60 days to fix it)... Maybe it is good idea add some accessible popup with example text and designed contacts link to this widget?
